src/pages/contact_information.tsx

```diff
@@ -0,0 +1,107 @@
+import { Card, Image, Text, Button, Group, CardSection } from "@mantine/core";
+
+interface ContactCardProps {
+	title: string;
+	operationTitle: string;
+	operationTime: string;
+	imgUrl: string;
+	email: string;
+}
+
+const imageUrl = "/assets/images/logos/registrar.png";
+
+const contactCards = [
+	{
+		imgUrl: imageUrl,
+		title: "Academic Support",
+		operationTitle: "Operation Hours:",
+		operationTime: "Mon-Fri 08:30 - 17.30",
+		email: "ug_academic@stamford.edu",
+	},
+	{
+		imgUrl: imageUrl,
+		title: "Registrar",
+		operationTitle: "Operation Hours:",
+		operationTime: "Mon-Fri 08:30 - 17.30",
+		email: "registrar.bkk@stamford.edu",
+	},
+	{
+		imgUrl: imageUrl,
+		title: "Library",
+		operationTitle: "Operation Hours:",
+		operationTime: "Mon-Fri 08:30 - 17.30",
+		email: "library@stamford.edu",
+	},
+	{
+		imgUrl: imageUrl,
+		title: "Finance",
+		operationTitle: "Operation Hours:",
+		operationTime: "Mon-Fri 08:30 - 17.30",
+		email: "cashier_rm9@stamford.edu",
+	},
+	{
+		imgUrl: imageUrl,
+		title: "IT Support",
+		operationTitle: "Operation Hours:",
+		operationTime: "Mon-Fri 08:30 - 17.30",
+		email: "support@stamford.edu",
+	},
+	{
+		imgUrl: imageUrl,
+		title: "Student Service",
+		operationTitle: "Operation Hours:",
+		operationTime: "Mon-Fri 08:30 - 17.30",
+		email: "international-service@stamford.edu",
+	},
+];
+
+const ContactCard = ({ title, operationTime, operationTitle, imgUrl, email }: ContactCardProps) => (
+	<Card className="mx-auto max-w-[17rem] rounded-lg p-10 md:p-10 lg:p-10">
+		<CardSection className="max-w-xs">
+			<Image className="h-full w-full rounded-lg" src={imgUrl} alt="Contact Img" />
+		</CardSection>
+
+		<CardSection className="font-open-sans mt-4 flex text-lg font-semibold text-white md:mt-6 lg:mt-8">
+			{title}
+		</CardSection>
+
+		<CardSection className="mt-2 flex items-center text-sm md:mt-4 md:text-base lg:mt-6">
+			{operationTitle}
+			<br />
+			{operationTime}
+		</CardSection>
+
+		<Card.Section className="mt-4">
+			<div className="w-full">
+				<a href={`mailto:${email}`}>
+					<Button className="h-10 w-full bg-slate-800 text-blue-300">Email</Button>
```

Comment:
We should just let the button be tbh, it's a primary action and this one is too muted.
You should use the light variant for secondary buttons, i.e. the copy mail should be light variant and this one should just be the blue color similar to quicklinks.